<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>虚拟传感器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/TCSIM_Color.svg</normaloff>:/TCSIM_Color.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="7" column="6">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_Dtm">
      <property name="text">
       <string>离线</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Std">
      <property name="text">
       <string>离线</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="1" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>ADC加噪</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QDoubleSpinBox" name="spinBox_WaveTemperature">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000100000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <double>0.008000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>目标温度</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QSpinBox" name="spinBox_WaveADC">
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_TargetTemp">
         <property name="text">
          <string>25</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>温度加噪</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>目标ADC</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>目标微调</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="comboBox">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>5025395</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4952012</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4879210</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4804070</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4729148</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4658251</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4582114</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4513357</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="spinBox_Target">
         <property name="maximum">
          <number>16777215</number>
         </property>
         <property name="singleStep">
          <number>500</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <number>4879210</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QComboBox" name="cbAnalyse_Dtm">
      <item>
       <property name="text">
        <string>data\r\n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$T=data ;\r\n</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="0" colspan="9">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>表达式</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>y = aX^4 + bX^3 cX^2 +dX + e</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>5</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>N-1系数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>x10^n</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>-993.579977</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>5.05346901</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>-4</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>9.27421342</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>-11</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>6.54944412</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>-18</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_UpdataFactor">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QComboBox" name="cbAnalyse_Std">
      <item>
       <property name="text">
        <string>data\r\n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>$T=data ;\r\n</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="3" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>PID</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_P">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <double>0.080000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_D">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>P</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>限幅</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBox_Limit">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_I">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="singleStep">
          <double>0.000001000000000</double>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinBox_Period">
         <property name="suffix">
          <string>ms</string>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>9999999</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>周期</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>当前：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>当前：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_Temperature">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>25</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_ADC">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>4879210</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>334</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
